cloudinit/sources/DataSourceOpenStack.py

```diff
@@ -16,8 +16,8 @@
 
 LOG = logging.getLogger(__name__)
 
+
```

Comment:
Please avoid unnecessary whitespace changes

Reply:
Removed that sneeky whitespace @holmanb !